codalab/apps/web/static/js/worksheet/table_item_interface.jsx

```diff
@@ -112,6 +113,20 @@ var TableRow = React.createClass({
         this.props.updateRowIndex(this.props.rowIndex);
     },
 
+    handleOptionButtonClick(event) {
+        event.nativeEvent.stopImmediatePropagation();
```

Comment:
We have a statement to close a context menu when there is a click in any area of window:
`document.onclick = ContextMenuMixin.closeContextMenu;`

Of course it is not default and we have to stop event from bubbling up the event chain to prevent closing context menu at the moment when dropdown button is clicked.